docs/_config.yml

```diff
@@ -1,21 +1,21 @@
 title: Standardizing Service Mesh Value Measurement
 email: community@layer5.io #this is also the email contact forms will go
-description: "Service Mesh Performance Specification (SMPS): Measure your service mesh performance using SMPS. SMPS provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments."
+description: "Service Mesh Performance (SMP): Measure your service mesh performance using SMP. SMP provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments."
 baseurl: "" # the subpath of your site, e.g. /blog
 
 author: Layer5 Authors
-twitter_username: smp_spec
-github_username:  l5io
+twitter_username: smp
+github_username: l5io
 
 twitter:
   card: summary
-  username: smp_spec
-  url: "https://twitter.com/smp_spec"
-  image: "https://smp-spec.io/assets/img/logo/smps.png"
+  username: smp
```

Comment:
`smp` will work as the twitter username here but the handle is @smp_spec.

Reply:
@chandrashritii, thanks for noticing.
The "_spc" referring to specification can be removed on twitter so @PluckyPrecious suggested this change.
Apparently twitter.com/smp is already taken, should I revert back the change?

Reply:
@chandrashritii, reverted back 👍